INSTALL.md

```diff
@@ -141,65 +141,62 @@ packages necessary for building .debs, so be prepared to type your password for
 
 <a name="osx" />
 ##OS X
+Before we start, be aware that this section had a large guide that succeeded in only confusing.
+I've thrown it entirely out for something actually useful and helpful.
 
-###OSX Easy Install
+Also, if you want to use qTox download it by clicking the download button on tox.im, compiling is a pain
+that I don't want you to ever deal with and your finished project will have a couple issues.
```

Comment:
@tux3 You can't actually develop a totally good to go copy that you can go and distribute like that, but I've made it more friendly.